src/Symfony/Bundle/WebServerBundle/Resources/router.php

```diff
@@ -30,7 +30,13 @@
     return false;
 }
 
-$script = getenv('APP_FRONT_CONTROLLER') ?: 'index.php';
+$script = getenv('APP_FRONT_CONTROLLER');
+
+if (isset($_ENV['APP_FRONT_CONTROLLER']) && false === $script) {
```

Comment:
this doesn't work when "E" is not listed in `variables_order`, which is the default on Ubuntu
note that this file is used only with `php -S` so we don't care behavior on Apache
I'd suggest to add `-dvariables_order=EGPCS` when calling `php -S`, and the we can get rid of the call to getenv completly